package.json

```diff
@@ -201,9 +201,9 @@
     "react-is": "^18.3.1",
     "@types/node": "^20.16.11"
   },
-  "packageManager": "pnpm@9.12.1",
+  "packageManager": "pnpm@9.15.1",
```

Comment:
Do you know what caused this change?

Reply:
I did :D It's 9.15.1 on `master` and I figured they should match, right?

Reply:
Hmm, I'm not sure 🤔
Maybe someone from @mui/code-infra can advise?

Reply:
I have no arguments against this. 🤷 
It goes out of sync because we stop bumping dependencies on anything other than `master`. 🤷

Reply:
> I did it mostly for convenience - It's annoying to switch the version of pnpm when releasing from master and v7.x

Are you using `corepack`?
The version should be picked up automatically if you have it installed.
But I agree that it could still end up annoying if the person doing the release does not have that particular outdated version released. 🙈 



> On another note - have you thought about stopping the releases for v7?

I don't think we've had this discussion yet. 🤔 
AFAIK, we keep releasing the current stable if there are any relevant bug fixes and/or localization changes while we are in the pre-release phase.
Once the current major changes—releasing prior versions becomes increasingly less frequent. It's mostly bigger bug fixes that we consider back-porting.